tofhir-server/src/main/scala/io/tofhir/server/endpoint/SchemaDefinitionEndpoint.scala

```diff
@@ -114,11 +114,17 @@ class SchemaDefinitionEndpoint(schemaRepository: ISchemaRepository, mappingRepos
     }
   }
 
-  private def getSchemaByUrl(url: String): Route = {
+  private def getSchemaByUrl(url: String, returnType: Option[String]): Route = {
     get {
       complete {
         service.getSchemaByUrl(url) map {
-          case Some(schemaDefinition) => StatusCodes.OK -> schemaDefinition
+          case Some(schemaDefinition) =>
+            returnType match {
+              case Some(SchemaFormats.SIMPLE_STRUCTURE_DEFINITION) =>
+                StatusCodes.OK -> schemaDefinition.fieldDefinitions.get
```

Comment:
It would be helpful to include an explanation here to clarify why we differentiate between the fieldDefinitions of a SchemaDefinition and the entire object.